serde/src/de/impls.rs

```diff
@@ -1934,6 +1935,39 @@ where
     }
 }
 
+macro_rules! nonzero_integers {
+    ( $( $T: ident, )+ ) => {
+        nonzero_integers!(@ $(::std::num::$T,)+);
```

Comment:
The @ trick doesn't work on 1.13

Reply:
I’ve verified that it does if I swap the order of the macro’s two variants/patterns.